iceoryx_posh/include/iceoryx_posh/internal/capro/capro_message.hpp

```diff
@@ -1,100 +1,102 @@
-// limitations under the License.
-
-#pragma once
-
-#include "iceoryx_posh/capro/service_description.hpp"
-
-namespace iox
-{
-namespace popo
-{
-struct ReceiverPortData;
-}
-
-namespace capro
-{
-/// @brief Enum for service message types which are used in CaPro for
-/// service-oriented communication
-enum class CaproMessageType : uint8_t
-{
-    NOTYPE = 0,
-    FIND,
-    OFFER,
-    STOP_OFFER,
-    SUB,
-    UNSUB,
-    ACK,
-    NACK,
-    PUB,
-    REQ,
-    RES,
-    PING,
-    PONG,
-    MESSGAGE_TYPE_END
-};
-
-constexpr int32_t MAX_ENUM_STRING_SIZE = 64;
-constexpr char CaproMessageTypeString[][MAX_ENUM_STRING_SIZE] = {"NOTYPE",
-                                                                 "FIND",
-                                                                 "OFFER",
-                                                                 "STOP_OFFER",
-                                                                 "SUB",
-                                                                 "UNSUB",
-                                                                 "ACK",
-                                                                 "NACK",
-                                                                 "PUB",
-                                                                 "REQ",
-                                                                 "RES",
-                                                                 "PING",
-                                                                 "PONG"};
-
-
-enum class CaproMessageSubType : uint8_t
-{
-    NOSUBTYPE = 0,
-    SERVICE,
-    EVENT,
-    FIELD
-};
-
-/// @brief C'tors for CaPro messages
-class CaproMessage
-{
-  public:
-    /// @brief C'tor for CaPro Message with no type and service description
-    /// @param m_type(NOTYPE)       Type that declare no message type
-    /// @param m_requestPort(0)     No port
-    /// @return                     Nothing
-    CaproMessage() noexcept;
-
-    /// @brief C'tor for CaPro Message with type, service description
-    /// @param f_type                    Message type
-    /// @param f_serviceDescription      Service Description
-    /// @param m_requestPort(0)          No port
-    /// @return                          Nothing
-    CaproMessage(CaproMessageType f_type,
-                 const ServiceDescription& f_serviceDescription,
-                 CaproMessageSubType f_subType = CaproMessageSubType::NOSUBTYPE,
-                 popo::ReceiverPortData* f_requestPort = nullptr) noexcept;
-
-    CaproMessageType m_type{CaproMessageType::NOTYPE};
-    CaproMessageSubType m_subType{CaproMessageSubType::NOSUBTYPE};
-    ServiceDescription m_serviceDescription;
-    /// @brief Null-Pointer for request-port with no specific type
-    popo::ReceiverPortData* m_requestPort{nullptr};
-};
-
-} // namespace capro
-} // namespace iox
+// Copyright (c) 2019 by Robert Bosch GmbH. All rights reserved.
```

Comment:
I guess this is marked as changed due to line ending changes?

Reply:
Yes.